projects/isy-angular-widgets/src/lib/input-char/components/multi-select-button/multi-select-button.component.spec.ts

```diff
@@ -97,6 +97,51 @@ describe('Unit Tests: MultiSelectButtonComponent', () => {
     component.setDisabledState(true);
     expect(component.disabled).toBeTrue();
   });
+
+  it('should initialize activeIndices as an empty array', () => {
```

Comment:
**Anmerkung für alle neuen Testfälle in dieser Datei:**
Die Beschreibung sollte eher fachlich sein sowas wie: it('toogles only one section at once"), die Information, dass dies mit einem activeIndices Attribut gelöst wurde ist dabei nicht relevant.